src/java.base/share/man/java.md

```diff
@@ -4011,34 +4011,34 @@ application is executed. One example is Class Data Sharing (CDS), as described a
 that parses classes ahead of time. AOT optimizations can improve the start-up and
 warm-up performance of Java applications.
 
-The Ahead-of-Time Cache (AOTCache) is a container introduced in JDK 24 for
-storing artifacts produced by AOT optimizations. The AOTCache currently contains
-Java classes and heap objects. The plans is to include other types of artifacts,
-such as execution profiles and compiled methods, in future JDK releases.
+The Ahead-of-Time Cache (AOT Cache) is a container introduced in JDK 24 for
+storing artifacts produced by AOT optimizations. The AOT cache currently contains
+Java classes and heap objects. In future JDK releases, the AOT cache may contain additional
+artifacts, such as execution profiles and compiled methods.
 
-An AOTCache is specific to a combination of the following:
+An AOT cache is specific to a combination of the following:
```

Comment:
of the following
+application configuration options/data/states/conditions:

(Or something specific like that.)

Reply:
Actually the AOT cache is more permissive with changes in command-line arguments. However, the three items below are absolute requirements that must be met. That's why I discuss the command-line option compatibility in the "loading of an AOT cache can fail for a number of reasons ..." section below.